### PR TITLE
test: improve test case

### PR DESCRIPTION
**Pull request by CCFenner**

### CODE REVIEW

File: src/piper.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ const fs = require('fs')
 
 async function run () {
   try {
+    const command = core.getInput('command', { required: true })
```
Comment (fwilhe) [inline src/piper.js]: I was not aware of this option. How does it relate to this flag? https://github.com/SAP/project-piper-action/blob/master/action.yml#L6

Comment (CCFenner) [reply src/piper.js]: I was not aware of the `yaml` 😉 

The required flag makes the `core.getInput` fail if the parameter is not set.

Comment (CCFenner) [reply src/piper.js]: I was not aware of the `yaml` 😉 

The required flag makes the `core.getInput` fail if the parameter is not set.

Comment (fwilhe) [reply src/piper.js]: but with the provided default (in the yaml file), I'm not sure how that can happen. And I think `help` is a sensible default.

Comment (fwilhe) [reply src/piper.js]: So I'm not sure about the benefit of this change, but also I don't see any drawbacks currently..

Comment (CCFenner) [reply src/piper.js]: As the yaml goes into the `dist/index.js` I guess we would need to test this with an integration test. This way we have a test next to the code. But on the other hand, why build it again if we already get it for free..

Comment (CCFenner) [reply src/piper.js]: Or lets see this as a fallback.

File: test/piper.test.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-jest.mock('@actions/core');
+//jest.mock('@actions/core');
```
Comment (fwilhe) [inline test/piper.test.js]: commented code by accident?

File: src/piper.js [hunk 3]
```diff
@@ -5,10 +5,12 @@ const fs = require('fs')
...
+    const flags = core.getInput('flags')
+
+    console.log('Loading piper as latest release')
```
Comment (fwilhe) [inline src/piper.js]: what is the purpose of this log? currently this is what always happens

**Pull request merged**